Game.java

```diff
@@ -0,0 +1,126 @@
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.util.Scanner;
+
+public class Game {
+	
+	public Game(){
+		try{
+			this.board = setBoard();
+		}
+		catch(FileNotFoundException e){
+			// no such file
+		}
+	}
+	
+	enum Creatures {
+		Pacman,
+		Ghost1, Ghost2, Ghost3, Ghost4,
+		Point,
+		Wall,
+		Null
+	}
+	
+	enum Directions {
+		RIGHT,
+		LEFT,
+		UP,
+		DOWN
+	}
+	
+	private Creatures[][] board = new Creatures[30][30];
+	int ghostNum;
+	
+	public Creatures[][] getBoard() {
+		return board;
+	}
+
+	public int getBoardDimensions(){
+		return board.length * 100 + board[0].length;
+	}
+	
+	public Creatures[][] setBoard() throws FileNotFoundException{

```

Comment:
What makes this method public?
Usually, when exposing such method, one should get the board file path as an argument. otherwise, make it private.
I would recommend separating board init from game logic...